src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
+var pathToHelloWorld = Path.Combine(MyWorkingDirectory(), "HelloWorld.cs");
+var pathToOutput = Path.Combine(MyArtifactsDirectory(), "HelloWorld.exe");
+
+ProcessResult result;
+
+var msToCompile = WalltimeMs(out result,
```

Comment:
For a local execution it would be more useful to just run csc.exe HelloWorld.cs  10 times and get average/best timings.

Perhaps we should have both kinds of measurements. Maybe even configurable. - total times, + with/without profiles.

It could be nice to have separate wallclock and CPU times too.